.github/workflows/static.yml

```diff
@@ -0,0 +1,26 @@
+name: Static Analysis
+
+on:
+  push:
+    paths:
+      - '**.php'
+      - 'phpstan.neon.dist'
+
+jobs:
+  phpstan:
+    name: PHPStan
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.2'
```

Comment:
Should we use 8.3 here?

Reply:
For PHPStan I tend to use the oldest supported PHP version. But either is fine.